compute/instances/troubleshooting/fix-dns-routed-ipv6-only-debian-bullseye.mdx

```diff
@@ -0,0 +1,143 @@
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [an Instance](/compute/Instances/how-to/create-an-Instance) running Debian Bullseye (11) with an IPv6-only setup
+</Message>
+
+<Message type="important">
+    This guide explains how to switch from the traditional `ifupdown` toolset to the modern `netplan`, which is used in recent Debian releases. The process is straightforward if your Instance is using the default network configuration provided by Scaleway metadata. 
+    However, if your Instance has a customized network setup, please ensure you are comfortable with both `ifupdown` and `netplan` before proceeding. You can safely apply this procedure to a newly created Instance.
+</Message>
+
+## Technical explanation
+
+When a Scaleway Instance uses routed IP addresses, the IPv6 stack is configured automatically using [`SLAAC`](https://datatracker.ietf.org/doc/html/rfc4862). With this method, the Instance is periodically advertised of various network configuration details, including the addresses of DNS servers that it should be using. The Instance is then free to consume these advertisements or not. By default, the operating system images provided by Scaleway are configured to leverage these advertisements to configure the IPv6 networking and the related DNS servers. The Debian Bullseye image is no exception.
+
+When configuring the network at boot time, the `cloud-init` software is in charge of detecting the appropriate network configuration method used by the system at hand, then to write and/or apply the needed configuration files/parameters according to this detected method. On Debian Bullseye, and because of [`cloud-init`'s built-in order of detection](https://cloudinit.readthedocs.io/en/latest/reference/network-config.html#network-output-policy), the primarily detected method is [ENI](https://cloudinit.readthedocs.io/en/latest/reference/network-config-format-eni.html), which configures the network through Debian's well known `/etc/network/interfaces*` set of files, along with the `ifupdown` toolset.
+
+For some reason, this configuration method does not seem to play nicely with SLAAC's DNS advertisements. This results in an absence of DNS resolver configuration, thus breaking most of the network activities.
+
+Due to its modern nature and active maintenance, [`netplan` is a favorable option for configuring cloud systems](https://www.debian.org/doc/manuals/debian-reference/ch05.en.html#_the_modern_network_configuration_for_cloud) and conveniently addresses the current issue. However, in alignment with our image building policy, which aims to minimize alterations to the official upstream cloud image, `netplan` is intentionally not enabled by default in Scaleway's Debian Bullseye image.
+
+## Checking if your Instance is concerned
+
```

Comment:
Maybe add a note about the requirement of having the CLI installed, and a link to a documentation about how to install it.